api/src/game_state_server.py

```diff
@@ -98,9 +99,9 @@ def new_connection_request(self, client_id: Hashable, room_id: str) -> Message:
             if tokens_to_load:
                 for token_data in tokens_to_load:
                     try:
-                        token = Token(**token_data)
-                    except TypeError:
-                        raise
+                        token = from_dict(data_class=Token, data=token_data)
+                    except (WrongTypeError, MissingValueError, TypeError) as e:
+                        print(e)
```

Comment:
have to also raise here

Reply:
should this be print_exc?

Reply:
Raising here didn't seem like the right decision because the rest of the room can still load, ignoring the bad token(s). Thoughts?

Reply:
Ah, I see. If your goal is partial recovery from corrupted data I think a comment here noting that would be valuable